inst/extdata/tests/math-basic.yml

```diff
@@ -0,0 +1,60 @@
+- abs:
```

Comment:
Is the name of the yml file important or needed? If not, I'd like to revisit having a single yml test file.  The reason is that it'll make it easier for us and other users to perform a text search for a specific code or test.

Reply:
We discussed multiple YAML files helping to segment tests.  It also integrates nicely with `testthat` since the `ListReporter` supports "file" as a parameter in the results output.  Find-in-files to the rescue if people do not know what file they are looking for!  😄